src/Compilers/CSharp/Portable/Binder/Semantics/Operators/UnaryOperatorOverloadResolution.cs

```diff
@@ -263,16 +260,26 @@ private void GetAllBuiltInOperators(UnaryOperatorKind kind, BoundExpression oper
                 operators.Add(pointerOperator.Value);
             }
 
-            CandidateOperators(operators, operand, results, ref useSiteDiagnostics);
+            CandidateOperators(operators, operand, results, ref useSiteDiagnostics, skipNativeIntegerOperators: !operand.Type.IsNativeIntegerOrNullableNativeIntegerType());
```

Comment:
>skipNativeIntegerOperators: !operand.Type.IsNativeIntegerOrNullableNativeIntegerType [](start = 84, length = 84)

Could you please move this computation into `CandidateOperators`?

Reply:
The `skipNativeIntegerOperators` check applies to built-in operators only and the caller can determine that more easily than the `CandidateOperators()` method.